h/security/policy/_auth_client.py

```diff
@@ -54,6 +54,9 @@ def handles(cls, request):
             ) in cls.API_WHITELIST
         return False
 
+    def forget(self, *_args, **_kwargs):
+        return []
```

Comment:
Adding `remember()` and `forget()` methods to `AuthClientPolicy` now that it no longer inherits these from `IdentityBasedPolicy`.